docs/deployment/gke/gcp-workload-identity.md

```diff
@@ -0,0 +1,72 @@
+# Granting GCP Workload Identity to Falcon Container Injector
+
+Falcon Container Injector may need [GCP Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)
+to read GCR or Artifact Registry. In many cases this GCP Workload Identity is assigned or inherited automatically. However, if you
+are seeing errors similar to the following you may need to follow this guide to assign the identity manually.
+
+```
+time="2022-01-14T13:05:11Z" level=error msg="Failed to handle webhook request" error="Failed to retrieve image details for \"gcr.io/\" in container in pod: Failed to get the image config/digest for \"gcr.io/" on \"eu.gcr.io\": Error reading manifest latest in gcr.io/: unauthorized: You don't have the needed permissions to perform this operation, and you may have invalid credentials. To authenticate your request, follow the steps in: https://cloud.google.com/container-registry/docs/advanced-authentication"
+```
+
+## Assigning GCP Workload Identity to Falcon Container Injector
+
+Conceptually, the following tasks need to be done in order to enable ECR pull from the injector
```

Comment:
typo: should be GCR or Artifact Registry

Reply:
Thanks!